src/ramses_tx/parsers.py

```diff
@@ -750,7 +751,7 @@ def parser_0418(payload: str, msg: Message) -> PayDictT._0418 | PayDictT._0418_N
         return null_result
 
     # NOTE: such payloads have idx=="00": if verb is I, can safely assume log_idx is 0,
-    # but for RP it is sentinel for null (we can't know the correspondings RQ's log_idx)
+    # but for RP it is sentinel for null (we can't know the corresponding RQ's log_idx)
```

Comment:
Do continue to point out any typos, etc. - but they should be separate from your PR - I will fix this & you can rebase your PR.